torch/_torch_docs.py

```diff
@@ -2845,6 +2845,36 @@ def merge_dicts(*dicts):
         tensor([False,  True,  False,  True,  False])
 """)
 
+add_docstr(torch.isposinf,
+           r"""
+isposinf(input, out=None) -> Tensor
+Returns a new tensor with boolean elements representing if each element is `+INF` or not.
+Args:
+    {input}
+    {out}
+Example::
+    >>> a = torch.tensor([-float('inf'), float('inf'), 1.2])
+    >>> a
+    tensor([  -inf,    inf, 1.2000])
+    >>> torch.isposinf(a)
+    tensor([False,  True, False])
+""".format(**common_args))
+
+add_docstr(torch.isneginf,
+           r"""
```

Comment:
See above comments.

Reply:
The docs of `isneginf` has been updated as well.

Reply:
I'm sorry, what I said was confusing. The portion of the docs I was referring to should be:

```
Args:
  {input}

Keyword args:
  {out}
```

Since `input` is a position argument but `out` is keyword-only.